### ISSUE TITLE
Left joy con has significant input delay

### ISSUE THREAD

**Issue by Ilmu011** [sam0x17/XJoy#58]
The left joy con has a significant input delay compared to the right joy con. It's so bad, that it renders the whole setup unplayable. It's very peculiar since the right joy con seems to have no input delay whatsoever.

**Comment by Ilmu011** [comment]
I looked into this a little more and found out that the joy con which has the input delay depends on which one I connected to my PC first. If I connect the right joy con first and then the left, the left joy con has an input delay. Do it the other way round and the right one has the input delay. In both cases, the other joy con has virtually no input delay, so I think the problem might lie within implementation.

**Comment by rainygg** [comment]
Updating the JoyCons firmware when connected to the Switch will possibly fix this. Give this a shot.

**Comment by Ilmu011** [comment]
@rainygg I went into the System Settings and performed a "controller update" while both joycons were connected. Is that what you meant? It did update them and says they are now up to date.

I went ahead and tested it again and the problem still remains though.

**Comment by sam0x17** [comment]
The pre-analog support versions of XJoy didn't have this issue, might have
luck trying one of the really old ones.

**Comment by Ilmu011** [comment]
@sam0x17 Ok, thanks for the suggestion. That would be v0.1.8, right?

**Comment by Ilmu011** [comment]
@sam0x17 Issue fixed! Under v0.1.8 I have no input delays whatsoever! Thanks for the suggestion!

**Comment by sam0x17** [comment]
Yeah anyone researching how to fix this permanently in the newer versions,
look at the diff between 0.1.8 and analog support changes the inefficiency
or concurrency bug or whatever was added there. I am a little unfamiliar
with how the analog stuff works but I reviewed it to the best of my ability
at the time.
